src/Map/CargoFormat/CargoFormat.php

```diff
@@ -11,7 +11,7 @@ class CargoFormat extends CargoDisplayFormat {
 
 	private \ParserOutput $parserOutput;
 
-	public function __construct( \OutputPage $output, \ParserOutput $parserOutput ) {
+	public function __construct( \OutputPage $output, $parserOutput ) {
```

Comment:
It'd be better if this just replicated the parent constructor's signature, I think:

```php
/**
 * @inheritdoc
 */
public function __construct( $output, $parser = null )
```

Reply:
Thanks, done z

Reply:
Actually I don't think so. Types are nice. Now the code is hard to understand. But whatever, not spending more time on this Cargo integration